pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -798,6 +802,9 @@ func (lrw *logicalReplicationWriterProcessor) flushChunk(
 ) (flushStats, error) {
 	batchSize := lrw.getBatchSize()
 
+	lrw.debug.RecordChunkStart()
```

Comment:
i could see a chunk size and chunk flush time histograms being useful as well.

Reply:
I think this is only really useful for crl engineers tuning it, not for operators to observe it, so it doesn't seem to justify the overhead of a metric that every single production user pays to maintain.